tools/workspace-plugin/src/plugins/workspace-plugin.spec.ts

```diff
@@ -323,6 +321,7 @@ describe(`workspace-plugin`, () => {
 
     it('should create default nodes for v9 stories project', async () => {
       await tempFs.createFiles({
+        'proj/stories/.storybook/main.js': '',
```

Comment:
plugin creates storybook target based on context, so if config would not exist storybook would not be generated.